### ISSUE TITLE
[Feature]: Disable desktop notifications if SimpleX Lock is enabled and locked.

### ISSUE THREAD

**Issue by GreyHorizon84** [simplex-chat/simplex-chat#5009]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Platform

Linux

### App version

6.0.5

### Feature

Currently on the Linux desktop version, if you have SimpleX Lock enabled and the inactivity period has been activated, notifications will still be sent as they are configured to be sent normally.

Since the spirit of the lock functionality is to prevent observers from accessing or reading messages while you are physically away, I think notifications should also be disabled, or at least censored, regardless of the notification settings.

I think it would be reasonable to simply disable notifications after locking, or just have a censored notification (without contact or message details). This could also be configured separately in the notifications tab.

**Comment by GreyHorizon84** [comment]
> you can configure notifications to not show username/content

Yes, but as it was said in the original comment, I think when the lock is enabled and activated, it must either be an alternative notification preference/setting or censored by default. I think it's entirely reasonable to want your notifications to be displayed Username/Content when your client is not locked, censored when it is locked.

**Comment by otonoton** [comment]
I would also like an option to disable notifications entirely on desktop.

**Comment by Narasimha-sc** [comment]
> I would also like an option to disable notifications entirely on desktop.

as a workaround you can mute all chats
